tests/UI/pages/FO/myAccount/orderHistory.js

```diff
@@ -80,6 +80,18 @@ class OrderHistory extends FOBasePage {
     return this.getTextContent(page, `${this.orderTableColumn(orderRow, 5)} span`);
   }
 
+  /**
+   * Is invoice visible on order history table row
+   * @param page {Page} Browser tab
+   * @param orderRow {number} Row number in orders table
+   * @param orderId {number} The id of the order
+   * @returns {Promise<boolean>}
+   */
+  isInvoiceVisible(page, orderRow = 1, orderId) {
+    return this.elementVisible(page, `${this.orderTableColumn(orderRow, 6)}`
+      + ` a[href*="pdf-invoice&id_order=${orderId}"]`);
```

Comment:
Can we add a `data-role` attribute instead of using href ? (And adding the link to selectors)

Reply:
Done thx :+1: